blog/introduction.md

```diff
@@ -165,18 +165,24 @@ component main = Task();
 
 Than she can generate R1CS by running the command:
 
-`circom task.circom --r1cs --wasm --sym --c --o build --O0 --p bls12381`
+```
+mkdir build
+circom task.circom --r1cs --wasm --sym -o build --O0 -p bls12381
+```
 
-This will generate a file (*task.r1cs*) which describes a R1CS in the *Cricom*. After that Alice will need to create a witness (vector $s$) file, but this time she will use a *SnarkJS* tool for doing this. First she needs to create a json file which will describe all private inputs in our circuits. In our case this is a very simple task, because our only input value is $x$ which is “*3*”. Input file (*input.json*) will look like this:
+This will generate a file (*build/task.r1cs*), which describes a R1CS in the *Cricom* and (*build/task.wasm*), which compiles circuit to WebAssembly. After that Alice will need to create a witness (vector $s$) file, but this time she will use a *SnarkJS* tool for doing this. First she needs to create a json file which will describe all private inputs in our circuits. In our case this is a very simple task, because our only input value is $x$ which is “*3*”. Input file (*input.json*) will look like this:
```

Comment:
After reading first sentence you can come to conclusion that file `task.wasm` compiles circuit to WebAssembly. Is this true?

Reply:
You are right, please check the changes